scaleway/resource_instance_server.go

```diff
@@ -421,17 +421,17 @@ func resourceScalewayInstanceServerRead(d *schema.ResourceData, m interface{}) e
 				rootVolume["delete_on_termination"] = true // default value does not work on list
 			}
 
-			d.Set("root_volume", []map[string]interface{}{rootVolume})
+			_ = d.Set("root_volume", []map[string]interface{}{rootVolume})
 		} else {
 			additionalVolumesIDs = append(additionalVolumesIDs, volume.ID)
 		}
 	}
-	d.Set("additional_volume_ids", additionalVolumesIDs)
+	_ = d.Set("additional_volume_ids", additionalVolumesIDs)
 
 	////
 	// Read server user data
 	////
-	allUserData, err := instanceAPI.GetAllServerUserData(&instance.GetAllServerUserDataRequest{
+	allUserData, _ := instanceAPI.GetAllServerUserData(&instance.GetAllServerUserDataRequest{
```

Comment:
I suppressed the error here, but it might be a better idea to return an error in case it's not nil)